pyvo/dal/sia2.py

```diff
@@ -156,7 +156,7 @@ class SIA2Service(DALService, AvailabilityMixin, CapabilityMixin):
     generally not notice that, though.
     """
 
-    def __init__(self, baseurl, session=None, check_baseurl=True):
+    def __init__(self, baseurl, capability_description=None, session=None, check_baseurl=True):
```

Comment:
same, though this may get the `*` after all (we have changed SIA2 a lot, so I might just pull the plug for this on in a sub #507 PR for v1.5)